torchao/kernel/intmm.py

```diff
@@ -37,8 +37,12 @@ def safe_int_mm(input: torch.Tensor, mat2: torch.Tensor) -> torch.Tensor:
         """
         # torch.compile path
         if dynamo_is_compiling() or "FakeTensor" in input.__repr__():
-            return out_dtype(torch.ops.aten.mm.default, torch.int32, input, mat2)
-
+            try:
```

Comment:
I dont understand this change, is it related to supporting fp8 with autoquant?

Reply:
autoquant with dynamic int8 quant breaks on H100, this is addressing that

Reply:
ohh okay so not directly related to this PR